webapp/cas-server-webapp-resources/src/main/resources/bootstrap.properties

```diff
@@ -22,6 +22,9 @@ spring.cloud.config.watch.delay=PT10S
 spring.cloud.config.fail-fast=true
 health.config.enabled=true
 
+# Turning off to suppress warning and b/c CAS doesn't rely on it
```

Comment:
Not sure about the location of this; Should this be here or in the application.properties file?